history.py

```diff
@@ -0,0 +1,30 @@
+from catalog import models as cmod
+
+class LastFiveMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+
+        # holds IDs
+        request.session.get('last_five', [])
```

Comment:
One other way to do this is request.session.get('last_five', '')
this will do the same thing

Reply:
Oh cool! Thanks!